fast-af/fast-checkout-woocommerce
# PINT-479 - Move fast admin css

# Description

This update moves the Fast admin CSS to a static `.css` file to be enqueued using WordPress best practices.

See: https://fastaf.atlassian.net/browse/PINT-479

# Testing instructions

1. Login to the [staging admin](https://fasttestdev.wpcomstaging.com/wp-admin/).
2. Go to the [Fast settings page](https://fasttestdev.wpcomstaging.com/wp-admin/admin.php?page=fast).
3. Verify that everything is rendering properly.

# Checks

@ilkerulutas @brikr this is ready for review.
